tests/chainer_tests/training_tests/extensions_tests/test_snapshot.py

```diff
@@ -142,7 +142,7 @@ def test_find_latest_snapshot(self):
         for file in files[:-1]:
             file = os.path.join(self.path, file)
             open(file, 'w').close()
-        time.sleep(10e-3)
+            time.sleep(10e-2)
```

Comment:
10e-2 = 0.1?
0.1 * 100 = 10 seconds seem a bit too long for a single test.
Why is it needed?

Reply:
All these feature uses timestamps to sort the snapshot files. To make sure files' mtimes are correctly sorted in filesystems which have 1-second resolution e.g. HFS+ or ext3, I put sleep at each iteration. For example, format like `trainer_snapshot_{.iteration}` and filenames like `trainer_snapshot_9` and `trainer_snapshot_99` need timestamp to sort out the latest one without using integer knowledge in the format. If you think it's too long, I can retry Travis test with `10e-3` seconds interval.

Reply:
I also of course understand that 10 seconds is too long for one test.  Will try 10e-3 second.